ui/raidboss/data/05-shb/raid/e6n.txt

```diff
@@ -7,13 +7,9 @@ hideall "--sync--"
 # -ii 4BDD 4BE1 4BEB 4BEA 4BE5 4BE9 4BEC 4BD9 4BE7 4E39 4C1D 4C20 4F98
 # -ic "Tumultuous Nexus" "Great Ball Of Fire"
 
-0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
-
-0.0 "--sync--" sync /:Engage!/ window 0,1
-
 # Garuda solo
-2.0 "--sync--" sync / 1[56]:[^:]*:Garuda:366:/ window 3,1
-13.5 "Ferostorm" # sync / 1[56]:[^:]*:Garuda:4BD[DEF]:/
+0.0 "--sync--" sync / 104:[^:]*:1($|:)/ window 0,1
+13.5 "Ferostorm" # sync / 1[56]:[^:]*:Garuda:4BD[DEF]:/ window 14,7
```

Comment:
This sync is commented out so this will not work.  I'm not sure why, I'd have to look at a log.

Reply:
> This sync is commented out so this will not work. I'm not sure why, I'd have to look at a log.

Not sure.  But there is still a long sync on the Superstorm right after, so it should still be okay?

Reply:
That sync has been commented out since the initial timeline was added in #1118; the only thing I could think of is that it was to prevent having two syncs on the two Ferostorm abilities that go off at the same time, but then the regex was updated to capture either one and the sync was left commented out.

There's also some massive drift here; on a test run I did today, the Ferostorm ability didn't come out until 31.0s, but the timeline has it at 13.5s.  I don't have any saved logs from when this was current content to see if it's always been like that.

Reply:
So the drift I'm seeing appears to be a bug with `make_timeline` and/or `test_timeline`, here are the first few relevant lines from the combat log (trimmed for space):
```
260|2023-06-21T02:49:01.6800000-04:00|0|1|
21|2023-06-21T02:49:16.1460000-04:00|40011872|Garuda|4BDF|Ferostorm
21|2023-06-21T02:49:16.6350000-04:00|4001187B|Garuda|4BDD|Ferostorm
22|2023-06-21T02:49:23.2620000-04:00|40011872|Garuda|4BD7|Superstorm
```

And here is an excerpt of what `make_timeline` gave when I tested against this log:

```
0.0 "--sync--" sync / 104:[^:]*:1($|:)/ window 0,1
31.0 "Ferostorm" sync / 1[56]:[^:]*:Garuda:4BDF:/
31.4 "Ferostorm" sync / 1[56]:[^:]*:Garuda:4BDD:/
38.1 "Superstorm" sync / 1[56]:[^:]*:Garuda:4BD7:/
```

The time given between the combat start line and the Ferostorm line is much larger than the log timestamps.

Reply:
Yeah, I'm not sure what's going on with e6n here.  Maybe we can break this out into a separate issue to try to figure what's going on (with the timeline or make_timeline?)

Reply:
New issue #5635 opened.  I'll remove the comment-out on the sync, since it looks like from the log @xiashtra posted that the first Ferostorm is roughly around the time where the sync should occur in the timeline.  Once I finish up with this PR, I'll try to find some time this weekend to dig in a little further on the bug.